README.md

```diff
@@ -1,13 +1,11 @@
 # Cluedo - Mystery Randomizer
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+So, our job was that build a mystery picker to find something or someone and pick randomly. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+The first problem to find out how to make functions what are working together and actually it was quite helpful that I grabbed paper and pen and draw/write how I imagine the problem and fix it. So sometime go back to the past and dont be afraid of using paper and pen.(:
```

Comment:
Love that you used paper and pen! That's a makes a really god overview - great idea! 👌🏻